internal/uvm/constants.go

```diff
@@ -34,4 +36,43 @@ const (
 var (
 	errNotSupported = errors.New("not supported")
 	errBadUVMOpts   = errors.New("UVM options incorrect")
+
+	// V5 GUIDs for SCSI controllers
+	// These GUIDs are created with namespace GUID "d422512d-2bf2-4752-809d-7b82b5fcb1b4"
+	// and index as names. For example, first GUID is created like this:
+	// guid.NewV5("d422512d-2bf2-4752-809d-7b82b5fcb1b4", []byte("0"))
+
+	ScsiControllerGuids = []guid.GUID{
```

Comment:
is this exported intentionally?

Reply:
Yeah, we need to use this in both hcsshim & opengcs.